<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/qi38tr.qrc">
    <normaloff>:/main/main/process.png</normaloff>:/main/main/process.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
      <property name="icon">
       <iconset resource="../res/qi38tr.qrc">
        <normaloff>:/main/main/page_process.png</normaloff>:/main/main/page_process.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Templates</string>
      </property>
      <property name="icon">
       <iconset resource="../res/qi38tr.qrc">
        <normaloff>:/main/main/templates.png</normaloff>:/main/main/templates.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QStackedWidget" name="settingsWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalPage">
      <widget class="QCheckBox" name="saveLocale">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>8</y>
         <width>355</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Remember selected language (User Interface)</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>50</y>
         <width>387</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>Google Translation</string>
       </property>
       <widget class="QComboBox" name="gLangFrom">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>42</y>
          <width>145</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="gLangTo">
        <property name="geometry">
         <rect>
          <x>192</x>
          <y>40</y>
          <width>175</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>20</y>
          <width>56</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>From</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>194</x>
          <y>16</y>
          <width>56</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="templatesPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QComboBox" name="templatesList"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Template</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="5" column="2" colspan="2">
          <widget class="QTextEdit" name="ignoreStr">
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;lt;script.*script&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;lt;!--.*--&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;//.*\n&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;//.*\n&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;/\*.*/&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Ignore</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;*Ignore &lt;span style=&quot; text-decoration: underline;&quot;&gt;comments&lt;/span&gt; in source code etc.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Searching for</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2" colspan="2">
          <widget class="QLineEdit" name="searchFor"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Template</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QComboBox" name="phpTemplatesList"/>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLineEdit" name="phpTemplName"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>File to save
 template</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2" colspan="2">
          <widget class="QTextEdit" name="saveTemplate"/>
         </item>
         <item row="8" column="2">
          <widget class="QPushButton" name="newPhpTemplate">
           <property name="text">
            <string>New template</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QPushButton" name="phpTemplSave">
           <property name="text">
            <string>Save Template</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Parsing language
 file</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QLineEdit" name="langFileRx"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>File extension</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QLineEdit" name="fileExt"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/qi38tr.qrc"/>
 </resources>
 <connections/>
</ui>
